static/institution.sql

```diff
@@ -69,12 +69,11 @@ ROW(
     3421 -- NULL
```

Comment:
Could you also replace this?

```suggestion
    NULL
```